From d0120fc59716fe7486f4682f99d17316e9d1d4b3 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Wed, 7 May 2025 11:03:52 +0200
Subject: [PATCH] arm64: dts: st: fix SPDX for all downstream stm32mp2x device
 trees

Use SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) for all
downstream stm32mp2x device trees.

Change-Id: I5951e6245e957d717b71e495ae203ec8dfd3d165
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/456292
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1-ca35tdcid-resmem.dtsi | 2 +-
 arch/arm64/boot/dts/st/stm32mp25xxaj-pinctrl.dtsi            | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1-ca35tdcid-resmem.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1-ca35tdcid-resmem.dtsi
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: (GPL-2.0-or-later OR BSD-3-Clause)
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
 /*
  * Copyright (C) STMicroelectronics 2022 - All Rights Reserved
  * Author: Loic Pallardy loic.pallardy@foss.st.com for STMicroelectronics.
--- a/arch/arm64/boot/dts/st/stm32mp25xxaj-pinctrl.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp25xxaj-pinctrl.dtsi
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: (GPL-2.0-or-later OR BSD-3-Clause)
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
 /*
  * Copyright (C) STMicroelectronics 2024 - All Rights Reserved
  * Author: Alexandre Torgue <alexandre.torgue@foss.st.com> for STMicroelectronics.
